src/test/java/org/ladocuploader/app/journeys/LaDigitalAssisterFlowJourneyTest.java

```diff
@@ -572,13 +572,13 @@ void fullDigitalAssisterFlow() {
     assertThat(testPage.getTitle()).isEqualTo(message("home-expenses.title"));
 
     testPage.clickElementById("householdHomeExpenses-Rent-label");
-    testPage.clickElementById("householdHomeExpenses-Other home expenses-label");
+    testPage.clickElementById("householdHomeExpenses-Other-label");
     testPage.clickContinue();
 
     assertThat(testPage.getTitle()).isEqualTo(message("home-expenses-amount.title"));
 
     testPage.enter("householdHomeExpenseAmount_wildcard_Rent", "500");
-    testPage.enter("householdHomeExpenseAmount_wildcard_Other home expenses", "100");
+    testPage.enter("householdHomeExpenseAmount_wildcard_Other", "100");
```

Comment:
Is this just the label or is the input name changed as well? Might need to revisit the pdf preparer

Reply:
It's been updated so that these values will appear as "householdHomeExpenseAmount_wildcard_otherHomeExpenses"